regolith/schemas.py

```diff
@@ -786,6 +803,73 @@
         'title': {'description': 'for example, Dr., etc.',
                   'required': False,
                   'type': 'string'}},
+    'presentations': {
+        '_description': {
+            'description': 'This collection describes presentations that group'
+                           'members make at conferences, symposia, seminars and'
+                           'so on.'},
+        '_id': {'description': 'unique id for the presentation',
+                'required': True,
+                'type': 'string'},
+        'abstract': {'description': 'abstract of the presentation',
+                     'required': False,
+                     'type': 'string'},
+        'authors': {
+            'description': 'Author list.  Each entry must be discoverable'
+                           'in the name or aka of a person',
+            'required': True,
+            'type': 'string'},
+        'begin_year': {'description': 'year the conference or trip begins.',
+                       'required': True,
+                       'type': 'integer'},
+        'begin_month': {'required': True,
+                        'type': 'string'},
+        'begin_day': {'required': False,
+                      'type': 'integer'},
+        'department': {'description': 'department of the institution where the'
+                                      'presentation will be made, if '
+                                      'applicable.  should be discoverable in '
+                                      'institutions.',
+                       'required': False,
+                       'type': 'string'},
+        'end_year': {'description': 'year the conference or trip ends',
+                     'required': False,
+                     'type': 'integer'},
+        'end_month': {'required': False,
+                      'type': 'string'},
+        'end_day': {'required': False,
+                    'type': 'integer'},
+    },
+    'institution': {'description': 'institution where the'
```

Comment:
Is this part of the presentation? If so it seems that the line above has an extra close `}` which is pushing institution into it's own collection.